packages/wonder-blocks-timing/src/util/__tests__/interval.test.js

```diff
@@ -131,12 +144,16 @@ describe("Interval", () => {
         it("should clear the active interval", () => {
             // Arrange
             const action = jest.fn();
-            const interval = new Interval(() => action(), 500);
+            const interval = new Interval(
+                () => action(),
+                500,
+                SchedulePolicy.OnDemand,
+            );
```

Comment:
Why the switch to `OnDemand` for this test?

Reply:
Because that was what the test was written to work with, but it didn't use it and so it fails.